lib/screens/v2/widget_instance/train_crowding.ex

```diff
@@ -0,0 +1,82 @@
+defmodule Screens.V2.WidgetInstance.TrainCrowding do
+  @moduledoc """
+  A widget that displays the crowding on a train that is en route to the current station.
+  """
+
+  alias Screens.Config.Screen
+  alias Screens.Config.V2.Triptych
+  alias Screens.Predictions.Prediction
+
+  defstruct screen: nil,
+            prediction: nil,
+            now: nil
+
+  @type t :: %__MODULE__{
+          screen: Screen.t(),
+          prediction: Prediction.t(),
+          now: DateTime.t()
+        }
+
+  @type widget_data :: %{
+          destination: String.t(),
+          arrival_time: String.t(),
+          crowding: list(crowding_data),
+          platform_position: number,
+          front_car_direction: :left | :right,
+          now: String.t()
+        }
+
+  @type crowding_data :: %{
+          crowding_level: 1 | 2 | 3,
+          percentage: number
```

Comment:
To make things super simple, we should be able to remove this. If I'm understanding correctly, we will always pick the crowding color on the frontend based on `crowding_level`. Should need `percentage` at all.

Reply:
Oh yeah! Seems so!